vizro-core/tests/unit/vizro/models/_controls/test_parameter.py

```diff
@@ -100,6 +100,16 @@ def test_set_categorical_selectors_with_missing_options(self, test_input):
         ):
             parameter.pre_build()
 
+    def test_set_datepicker_values_invalid(self):
+        parameter = Parameter(targets=["scatter_chart.x"], selector=vm.DatePicker())
```

Comment:
bit of a random one no? This is essentially testing the Selector again, and not the parameter no? We may actually have some tests like that here, so do not mind keeping it, just curious what you think

Should we not rather only test if the whoe parameter action works with a datepicker as selector?

Reply:
This test actually tests a newly added method `_set_date_picker_values` inside the Parameter model. 

I'm not entirely convinced about the necessity of testing the date picker as an input for the parameter action. This is because the parameter action is already tested and its functionality doesn't rely on the input type. The input is just propagated to the target without significant impact.

Reply:
Ok that makes sense!